tslib/storage/src/journal.ts

```diff
@@ -340,7 +340,31 @@ const loadJournalStorage = (arrayBuffer: ArrayBuffer): Optuna.Study[] => {
     if (log === "") {
       continue
     }
-    const parsedLog: JournalOpBase = JSON.parse(log)
+
+    const parsedLog: JournalOpBase = (() => {
```

Comment:
Could you add some tests to check this change in the following pull request?
https://github.com/optuna/optuna-dashboard/blob/main/tslib/storage/test/journal.test.mjs